### DISCUSSION TITLE
KeePassXC-Browser 1.8.4 / KeePassXC 2.7.4 in Firefox on Ubuntu 22 only returns entries from currently selected database

### DISCUSSION THREAD

**Discussion by RayCulp** [keepassxreboot/keepassxc#9193] [Q&A]
Hi everyone,

I have two databases open and have activated the option "Search in all opened databases for matching credentials", but KeePassXC-Browser only shows me the credentials from the database in the currently selected tab. Switching tabs allows me to access the credentials in the other database, but requires access permission each time. Any idea why this is happening? Thanks!

Ray

**Comment by varjolintu** [comment]
The active database must be connected to KeePassXC. We are currently improving how these situations are handled. You can follow the process here: https://github.com/keepassxreboot/keepassxc-browser/issues/1275

**Comment by RayCulp** [comment]
Thank you very much for that information. 

It turns out that the second database is a copy of the first, and setting a new  Root Group ID fixed the problem. The extension now offers credentials from both databases. 

![image](https://user-images.githubusercontent.com/7621330/222452718-96d9870e-6301-42f6-adb5-cbd55fa71ab7.png)

It might be a good idea to add this information to the Wiki, perhaps a new page titled something like "Working with multiple open databases" or "Accessing credentials from multiple open databases".

**Reply by varjolintu** [reply]
@RayCulp I'm already working with a feature that will inform the user if two databases are detected with an identical Root Group ID.

**Reply by RayCulp** [reply]
@varjolintu Ah, excellent!
